Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/Faq.rst

```diff
@@ -45,6 +45,24 @@ of the extension or the :ref:`crowdin-initiative`.
 
    The language needs to be supported by TYPO3 itself as well, see :ref:`i18n_languages` for a list of all languages.
 
+.. _crowdin-faq-language-xlf-format:
+
+How to convert to the new language xlf file format
+--------------------------------------------------
+If you are downloading a xlf file from the Pootle language server or an old version of an extension,
+then it does not have the correct format. You can use a linux tool or a sophisticated editor to convert
```

Comment:
"... does not have the correct format ..." -> lacks description of what is the correct format.

Maybe better: "... does not have a "resname" attribute for each "trans-unit" tag. This is required for the display of the key in Crowdin when translating." (or something like that).